probe/endpoint/reporter.go

```diff
@@ -214,3 +235,29 @@ func (r *Reporter) makeEndpointNode(namespaceID string, addr string, port uint16
 func newu64(i uint64) *uint64 {
 	return &i
 }
+
+// procParsingSwitcher make sure that if eBPF tracking is enabled,
+// connections coming from /proc parsing are only walked once.
+func (r *Reporter) procParsingSwitcher() {
+	if r.conf.WalkProc && r.conf.UseEbpfConn {
+		r.conf.WalkProc = false
+		r.ebpfTracker.initialize()
```

Comment:
This is not really an initialization, we use it to signal when the initial data from proc parsing was fed to the ebpf tracker so it can take over.

The name is rather unfortunate and we'll try to address this with the redesign of the ebpf tracker.